docs/docs/winning-over-stakeholders.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: "Winning Over Stakeholders"
+title: "Conquistar los accionistas"
 ---
 
-Any website project has multiple stakeholders, such as developers, marketers, content creators, engineering leaders, executives, and clients.
+Cualquier proyecto del sitio web tiene varias accionistas, como los desarrolladores, los comercializadores, creadores de contenido, líderes de ingeniería, los ejecutivos, y los clientes.
 
-In smaller projects and organizations, one person may wear multiple hats. As a freelancer, you may be the developer, marketer, content creator, and engineering leader rolled into one. Working at a mid-sized agency or organization, you are likely to only wear one hat at a time.
+En proyectos y organizaciones más pequeñas, una persona puede usar muchos sombreros. Como un freelance, puede ser el desarrollador, el comercializador, el creador de contenido, y líder de ingeniería rodó en una. Trabajando en una agencia o organización de tamaño mediano, usted es probable para usar sólo un sombrero a la vez.
```

Comment:
Looks good, but I'd change:
"`(...) Trabajando en una agencia o organización de tamaño mediano, usted es probable para usar sólo un sombrero a la vez`".
to:
"`(...) Trabajando en una agencia u organización de tamaño medio, es probable que utilices sólo un sombrero a la vez`".